app/client/src/routes/home.tsx

```diff
@@ -468,159 +514,209 @@ function FilterFieldInputs({
   staticOptions,
 }: FilterFieldInputsProps) {
   // Store each field's element in a tuple with its key
-  const fieldsJsx: Array<[JSX.Element, string]> = removeNulls(
-    fields.map((fieldConfig) => {
-      const sourceFieldConfig =
-        'source' in fieldConfig &&
-        (fieldConfig.source as string) in sourceFields
-          ? sourceFields[fieldConfig.source as string]
-          : null;
-
-      const tooltip =
-        fieldConfig.label in glossary
-          ? glossary[fieldConfig.label].definition
-          : null;
-
-      switch (fieldConfig.type) {
-        case 'multiselect':
-        case 'select':
-          if (
-            !sourceFieldConfig &&
-            fieldConfig.type === 'multiselect' &&
-            fieldConfig.key in staticOptions &&
-            staticOptions[fieldConfig.key].length <= 5
-          ) {
+  const fieldsJsx: Array<[JSX.Element, string, string | undefined]> =
+    removeNulls(
+      fields.map((fieldConfig) => {
+        const sourceFieldConfig =
+          'source' in fieldConfig &&
+          (fieldConfig.source as string) in sourceFields
+            ? sourceFields[fieldConfig.source as string]
+            : null;
+
+        const tooltip =
+          fieldConfig.label in glossary
+            ? glossary[fieldConfig.label].definition
+            : null;
+
+        switch (fieldConfig.type) {
+          case 'multiselect':
+          case 'select':
+            if (
+              !sourceFieldConfig &&
+              fieldConfig.type === 'multiselect' &&
+              fieldConfig.key in staticOptions &&
+              staticOptions[fieldConfig.key].length <= 5
+            ) {
+              return [
+                <Checkboxes
+                  key={fieldConfig.key}
+                  label={fieldConfig.label}
+                  onChange={
+                    filterHandlers[fieldConfig.key] as OptionInputHandler
+                  }
+                  options={staticOptions[fieldConfig.key]}
+                  selected={
+                    (filterState[fieldConfig.key] as MultiOptionState) ?? []
+                  }
+                  className="margin-top-2"
+                  tooltip={tooltip}
+                />,
+                fieldConfig.key,
+                fieldConfig.size,
+              ];
+            }
+
+            const sourceKey = sourceFieldConfig?.key ?? null;
+            const sourceValue = sourceFieldConfig
+              ? sourceState[sourceFieldConfig.id]
+              : null;
+            const selectProps = {
+              apiKey,
```

Comment:
```suggestion
              additionalOptions: 'additionalOptions' in fieldConfig ? fieldConfig.additionalOptions : [],
              apiKey,
```

This should fix the issue of `Any` not being in the list.

Reply:
Thanks, I'm not sure how that got lost.